frontend/src/components/form/inputs/Interpreter.tsx

```diff
@@ -111,6 +111,7 @@ export default function InterpreterInputs () {
 
       <StyledField name='supplier' label='Supplier #' />
       <StyledField name='gst' label='GST' />
+      <StyledField name='criminalRecordCheck' label='Recrod Check Received' />
```

Comment:
typo

Reply:
`Recrod`

Reply:
Will this auto-type it to date? Or is it a text field?

Reply:
it is a text field as they gave us string dates that have no format